packages/govuk-frontend/src/govuk/all.mjs

```diff
@@ -103,6 +104,7 @@ export {
   Checkboxes,
   ErrorSummary,
   ExitThisPage,
+  GOVUKFrontendError,
```

Comment:
Only export our base `GOVUKFrontendError` rather than all error classes with the aim of:

- allowing separating our own errors from native ones in bulk, which users can do running `instanceof GOVUKFrontendError`
- but not crowding our export with each error we throw, as users can check the type of error through [their `name` property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name)

Reply:
Fab work in this PR Romaric

Do we want to discuss if we're ready for errors to be part of our "public" API?

Bit like the **I18n** class, we chose not to export it

(Gives us freedom to keep iterating without breaking changes)

Reply:
Let's default to not exporting it yet, can always add it in another PR